### ISSUE TITLE
WebIF: Make tabs (count) configurable according configured plugin functionality

### ISSUE THREAD

**Issue by sisamiwe** [smarthomeNG/smarthome#490]
Hi,

for Plugins offering divers functionality like AVM (having TR-064, Call-Monitor and AHA-Interface) or AVdevice (having zones) it would make sense to allow WebIF tabs also to be visible or hidden according configured pluign functionality.

Currently the tabcount for the WebIF can be configured, but you cannot leave a configured tab out at rendering.

I would like to propose to add a feature, that defined tabs can be left out at rendering. This might be done by not setting a tabtitle.
At the end, the tabtiltle could be set or not by using (if....endif) depending on the used plugin functionality.

Example:
The tab for callmonitor of the AVM Plugin could be visible or hidden in the WebIF depending wether the callmonitor is activated in the plugin or not.

`{% if callmonitor %}
    {% set tab5title = _("<strong>"'Call Monitor Items'"</strong> (" ~ call_monitor_item_count ~ ") ") %}
{% endif %}`

**Comment by onkelandy** [comment]
I think the best approach would be to have a specific name for hidden tabtitles to not break the current behvaiour of automatic tabtitle replacemt.
E.g.
```
{% if tabcount > 5 and tab6title != "hidden" %}
```

So the code in the plugin should be
```
{% if callmonitor %} {% set tab5title = _("<strong>"'Call Monitor Items'"</strong> (" ~ call_monitor_item_count ~ ") ") %} 
{% else %}
{% set tab5title = "hidden" %} 
{% endif %}
```

**Comment by msinn** [comment]
resolved by #491

**Comment by msinn** [comment]
SmartHomeNG v1.9.2 with the resolution has been released
